net_handler/message.c

```diff
@@ -109,7 +108,6 @@ void send_device_data(int dawn_socket_fd, uint64_t dawn_start_time) {
 
     //get information
     get_catalog(&catalog);
-    get_sub_requests(sub_map, NET_HANDLER);
```

Comment:
We need to determine the bitmap passed into `device_read_uid()`. The bitmap should have bits on only for parameters that 1) exist and 2) are readable. An inefficient way to do this is to iterate through every single device and get this information from `runtime_util.h` functions.

Reply:
Update: we are discussing what to do about this on Slack; will post an update once this has been resolved.

Reply:
The solution we came up with is using a `get_readable_param_bitmap()` util function.